src/poke_env/player/env_player.py

```diff
@@ -76,127 +67,23 @@ def __init__(
             Defaults to None.
         :type team: str or Teambuilder, optional
         """
-        super(EnvPlayer, self).__init__(
+        self._reward_buffer = {}
+        self.opponent_lock = Lock()
+        self.opponent: Optional[Union[Player, str]] = opponent
```

Comment:
If no opponent is passed, we should default to a `RandomPlayer` instance playing the same format.

Reply:
I agree, will push that on the next batch